src/components/solver/ActiveSolverTable/index.tsx

```diff
@@ -0,0 +1,280 @@
+      img {
+        padding: 0;
+      }
+    }
+  }
+  ${media.mobile} {
+    > thead > tr {
+      display: none;
+
+      > th:first-child {
+        padding: 0 1rem;
+      }
+    }
+    > tbody > tr {
+      grid-template-columns: none;
+      border: 0.1rem solid ${({ theme }): string => theme.tableRowBorder};
+      box-shadow: 0px 4px 12px ${({ theme }): string => theme.boxShadow};
+      border-radius: 6px;
+      margin-top: 10px;
+      padding: 12px;
+      &:hover {
+        background: none;
+        backdrop-filter: none;
+      }
+
+      td:first-child {
+        padding: 0 1rem;
+      }
+    }
+    tr > td {
+      display: flex;
+      flex: 1;
+      width: 100%;
+      justify-content: space-between;
+      margin: 0;
+      margin-bottom: 18px;
+      min-height: 32px;
+      span.span-inside-tooltip {
+        align-items: flex-end;
+        flex-direction: column;
+        img {
+          margin-left: 0;
+        }
+      }
+    }
+    > tbody > tr > td,
+    > thead > tr > th {
+      :nth-child(4),
+      :nth-child(5),
+      :nth-child(6),
+      :nth-child(7),
+      :nth-child(8) {
+        justify-content: space-between;
+      }
+    }
+    .header-value {
+      flex-wrap: wrap;
+      text-align: end;
+    }
+    .span-copybtn-wrap {
+      display: flex;
+      flex-wrap: nowrap;
+      span {
+        display: flex;
+        align-items: center;
+      }
+      .copy-text {
+        display: none;
+      }
+    }
+  }
+  overflow: auto;
+`
+
+const HeaderTitle = styled.span`
+  display: none;
+  ${media.mobile} {
+    font-weight: 600;
+    align-items: center;
+    display: flex;
+    margin-right: 3rem;
+    svg {
+      margin-left: 5px;
+    }
+  }
+`
+const HeaderValue = styled.span<{ captionColor?: 'green' | 'red1' | 'grey' }>`
+  color: ${({ theme, captionColor }): string => (captionColor ? theme[captionColor] : theme.textPrimary1)};
+  ${media.mobile} {
+    flex-wrap: wrap;
+    text-align: end;
+  }
+`
+const IdenticonWrapper = styled.div`
+  display: flex;
+  flex: 1;
+  align-items: center;
+  gap: 1rem;
+`
+
+export type Props = StyledUserDetailsTableProps & {
+  solvers: Solver[] | undefined
+  tableState: TableState
+}
+
+interface RowProps {
+  index: number
+  solver: Solver
+}
+
+const RowSolver: React.FC<RowProps> = ({ solver }) => {
+  const { id, name, address, numberOfTrades, numberOfSettlements, solvedAmountUsd } = solver
+  const network = useNetworkId()
+
+  if (!network) {
+    return null
+  }
+
+  return (
+    <tr key={id}>
+      <td>
+        <HeaderTitle>Name</HeaderTitle>
+        <HeaderValue>
+          <IdenticonWrapper>
+            <Identicon address={address} size="md" />
+            <BlockExplorerLink type="address" networkId={network} identifier={address} label={name} />
+          </IdenticonWrapper>
+        </HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>Trades</HeaderTitle>
+        <HeaderValue>{numberOfTrades}</HeaderValue>
```

Comment:
Could we have a formatter here? Same as `numberOfSettlements` down below, I think a thousands separator is important